src/lib/StorageFileApi.ts

```diff
@@ -332,4 +331,8 @@ export class StorageFileApi {
   _getFinalPath(path: string) {
     return `${this.bucketId}/${path}`
   }
+
+  _removeEmptyFolders(path: string) {
+    return path.replace(/^\/|\/$/g, '\1').replace(/\/\//g, '/')
```

Comment:
Should `'\1'` be `''` instead?

`/\/\//g` could be `/\/+/g` to handle the edge cases of more than two `/`.

Reply:
Thanks for the review @thebengeu , I've made the requested changes